### PR TITLE
find-doc functionality. Solves #80.

### PR DESCRIPTION
**Pull request by tomasz-biernacki**
As in the title

### CODE REVIEW

File: src/cljs/replumb/repl.cljs [hunk 1]
```diff
@@ -47,15 +47,22 @@
   "Given a namespace return all the public var analysis maps. Analagous to
   clojure.core/ns-publics but returns var analysis maps not vars."
   ([ns]
-   (ns-publics env/*compiler* ns))
-  ([state ns]
    {:pre [(symbol? ns)]}
    (->> (merge
-         (get-in @state [:cljs.analyzer/namespaces ns :macros])
-         (get-in @state [:cljs.analyzer/namespaces ns :defs]))
+         (get-in @st [:cljs.analyzer/namespaces ns :macros])
```
Comment (arichiardi) [inline src/cljs/replumb/repl.cljs]: Woah! Thanks!

File: src/cljs/replumb/repl.cljs [hunk 2]
```diff
@@ -47,15 +47,22 @@
...
+         (get-in @st [:cljs.analyzer/namespaces ns :defs]))
         (remove (fn [[k v]] (:private v)))
         (into {}))))
 
+(defn ns-interns
```
Comment (arichiardi) [inline src/cljs/replumb/repl.cljs]: Is this working at the repl?

Comment (arichiardi) [inline src/cljs/replumb/repl.cljs]: I tried, it works but not its docstring.

Comment (tomasz-biernacki) [inline src/cljs/replumb/repl.cljs]: Yes, it does not work but we need to override Clojure's `ns-interns` functionality.

**Comment by arichiardi** [comment]
Great, will merge this

**Pull request merged**